app/build.gradle

```diff
@@ -25,7 +25,7 @@ android {
         targetSdkVersion 28
         versionCode 11 // This versionCode is "frozen" for local builds. For "release" builds we
         // override this with a generated versionCode at build time.
-        versionName "3.2.2"
+        versionName "3.2.6.1"
```

Comment:
Our releases tab is getting pretty hard to read.  Suggest updating this to include pre-release metadata as per https://semver.org/#spec-item-9 (e.g., `3.2.6-LAT.1`).